draft-janfred-radext-radiusdtls-bis.md

```diff
@@ -607,23 +607,10 @@ Non-compliant, or unexpected packets will be ignored by the DTLS layer.[^proxymi
 [^src_7360_5_2]: Source: RFC7360, Section 5.2 with modifications
 
 RADIUS/DTLS clients SHOULD use PMTU discovery {{!RFC6520}} to determine the PMTU between the client and server, prior to sending any RADIUS traffic.
-Once a DTLS session is established, a RADIUS/DTLS client SHOULD use DTLS Heartbeats {{RFC6520}} to determine connectivity between the two systems.
-RADIUS/DTLS clients SHOULD also use the application-layer watchdog algorithm defined in {{RFC3539}} to determine server responsiveness.
-The Status-Server packet defined in {{RFC5997}} SHOULD be used as the "watchdog packet" in any application-layer watchdog algorithm.[^doublespec]{:jf}
-
-[^doublespec]: The Status-Server spec was already mentioned above. Maybe remove it from here?
 
 RADIUS/DTLS clients SHOULD proactively close sessions when they have been idle for a period of time.
-Clients SHOULD close a session when the DTLS Heartbeat algorithm indicates that the session is no longer active.
 Clients SHOULD close a session when no traffic other than watchdog packet and (possibly) watchdog responses have been sent for three watchdog timeouts.
-This behavior ensures that clients do not wast resources on the server by causing it to track idle sessions.
-
-When a client fails to implement both DTLS Heartbeats and watchdog packets, it has no way of knowing that a DTLS session has been closed.
-Therefore, there is the possibility that the server closes the session without the client knowing.
-When that happens, the client may later transmit packets in a session, and those packets will be ignored by the server.
-The client is then forced to time out those packets and then the session, leading to delays and network instabilities.
```

Comment:
This consideration is now void, since we have the requirement, that all peers implement the watchdog algorithm.
Maybe it would be helpful to add some text of this into "Design Decisions" as rationale or heads-up for implementers why the requirement is important.